src/bosh_azure_cpi/lib/cloud/azure/logger.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class CPILogger
+    def self.get_logger(device)
+      logger = Logging.logger(device,
+                              pattern: '%.1l, [%d #%p #%t] %l --%X{flag}: %m\n', # flag is a placeholder for request id
+                              date_pattern: '%Y-%m-%dT%H:%M:%S.%6N')
+      logger.level = :debug
```

Comment:
shall we read the logger level from config file?

Reply:
there is no config file for this param